test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/BulkUpdateE2ETests.cs

```diff
@@ -0,0 +1,892 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="BulkUpdateE2ETests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using AstoriaUnitTests.Tests;
+using FluentAssertions;
+using Microsoft.OData.Client.Materialization;
+using Microsoft.OData.Edm;
+using Microsoft.OData.Edm.Csdl;
+using Xunit;
+
+namespace Microsoft.OData.Client.TDDUnitTests.Tests
+{
+    /// <summary>
+    /// BulkUpdate E2E tests.
+    /// </summary>
+    public class BulkUpdateE2ETests

```

Comment:
Add tests with `.Expand`

`var customer = context.Customers.Expand("Orders").FirstOrDefault();`

Reply:
I have simulated responses that are expanded. I don't need to explicitly add .Expand()